shared/applications/dotnet/aspnetcore-and-mongodb/src/AspNetCoreExample/AspNetCoreExample.csproj

```diff
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.0.0" />
-    <PackageReference Include="MongoDB.Driver" Version="2.9.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.10" />
+    <PackageReference Include="MongoDB.Driver" Version="2.18.0" />
```

Comment:
Main reason for a bump was MongoDB.Driver. Old versions are not supported now https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/b1dc33c19e04ad7c626e95572637decfd44aede7/CHANGELOG.md#removed